client-hc/src/main/java/com/graphhopper/api/WebHelper.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *  Licensed to GraphHopper and Peter Karich under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for 
+ *  additional information regarding copyright ownership.
+ * 
+ *  GraphHopper licenses this file to you under the Apache License, 
+ *  Version 2.0 (the "License"); you may not use this file except in 
+ *  compliance with the License. You may obtain a copy of the License at
+ * 
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.api;
+
+import com.graphhopper.util.PointList;
+
+import java.net.URLEncoder;
+
+/**
+ * Code which handles polyline encoding and other web stuff.
+ * <p>
+ *
+ * @author Peter Karich
+ */
+public class WebHelper {
```

Comment:
Is this class now duplicate as we also have com.graphhopper.http.WebHelper and if yes, can we avoid this somehow?

Reply:
Yes and yes, in the next step, where we move all the Data Transfer Objects (including, probably, GHRequest / GHResponse) to _another_ module (without HTTP-client dependencies), which is then included by core, client _and_ server.